### ISSUE TITLE
Suppress Sendable protocol warnings

### ISSUE THREAD

**Issue by Etsuwo** [uber/mockolo#252]
I want to create a mock from a protocol that inherits from `Sendable` with this OSS, but I have a problem.

Mocks generated by this OSS have variables, so if I generate a mock from a protocol that inherits from `Sendable`, I get a warning as shown below.

```swift
/// @mockable
protocol Hoge: Sendable {
    func hoge() -> Int
}
```
↓ generated

```swift
final class HogeMock: Hoge {
    init() { }

    private(set) var hogeCallCount = 0 // warning: Stored property 'hogeCallCount' of 'Sendable'-conforming class 'HogeMock' is mutable
    var hogeHandler: (() -> (Int))?
    func hoge() -> Int {
        hogeCallCount += 1
        if let hogeHandler = hogeHandler {
            return hogeHandler()
        }
        return 0
    }
}
```

I want to suppress unnecessary warnings because they reduce visibility and make it difficult to notice necessary warnings. For this reason, I would like to suggest that mocks generated from protocols that inherit from Sendable be marked with `@unchecked sendable`.

Thank you.

**Comment by Etsuwo** [comment]
Sorry, I forgot to post my environment.

Mac OS version: Ventura 13.5.2
Xcode version : 15.0.1
Swift version : 5.9
mockolo version: 2.0.1

**Comment by sidepelican** [comment]
available on 2.1.1

**Comment by Etsuwo** [comment]
Thank you for your response! I will try the latest version.
